frontend/src/metabase/list-view/components/ListView/ListViewConfiguration.tsx

```diff
@@ -406,3 +413,32 @@ export const ListViewConfiguration = ({
     </DndContext>
   );
 };
+
+function generatePreviewSample(rows: RowValues[], cols: Lib.ColumnMetadata[]) {
```

Comment:
Showing fake values here could potentially scare or at least confuse the user.
Did we consider displaying some sort of placeholders instead?